x-pack/plugins/rollup/server/lib/search_strategies/rollup_search_request.ts

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-const SEARCH_METHOD = 'rollup.search';
-
-interface Search {
-  index: string;
-  body: {
-    [key: string]: any;
-  };
-}
-
-export const getRollupSearchRequest = (AbstractSearchRequest: any) =>
-  class RollupSearchRequest extends AbstractSearchRequest {
-    async search(searches: Search[]) {
-      const requests = searches.map(({ body, index }) =>
-        this.callWithRequest(SEARCH_METHOD, {
-          body,
-          index,
-          rest_total_hits_as_int: true,
-        })
```

Comment:
Since this endpoint was replaced with the `elasticsearch.client.transport.request` in `x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts`, the TSVB Table visualization fails to load. The reason is the next elasticsearch error logged: 
```
[error][data][elasticsearch] [illegal_argument_exception]: Unable to translate aggregation tree into Rollup.  Aggregation [pivot] is of type [FilterAggregationBuilder] which is currently unsupported.
```

It seems, we currently can't use the transport request in tsvb rollup indexes (at least for table aggs).
My suggestion is to implement own `search` method for `rollup_search_strategy.ts` and use `elasticsearch.client.asCurrentUser.rollup.rollupSearch` instead.